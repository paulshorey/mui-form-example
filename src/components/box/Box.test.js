import React from 'react';
import renderer from 'react-test-renderer';

// shallow + redux
import { shallow, connected, shallowConnected } from 'index.testing';
import { Box } from './Box';

it('renders correctly', () => {
	const tree = renderer
		.create(
			shallowConnected(
				<Box
					box={{
						theme: '',
						title: '',
						icon: '',
						Children: <div>hello</div>,
						TopChildren: <span />,
					}}
				/>
			)
		)
		.toJSON();
	expect(tree).toMatchSnapshot();
});
