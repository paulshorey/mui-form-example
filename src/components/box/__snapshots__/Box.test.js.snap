// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="box  sc-bdVaJa eujpTF"
  id=""
>
  <div
    className="box_content"
  >
    <div
      className="box_top"
    >
      <div
        className="items"
      >
        <div
          className="item title"
        >
          
          
        </div>
        <span />
      </div>
    </div>
    <div
      className="box_middle"
    >
      <div>
        hello
      </div>
    </div>
  </div>
</div>
`;
